oracle/scripts/getValidatorStartBlocks.js

```diff
@@ -33,12 +33,20 @@ async function getStartBlock(rpcUrl, bridgeAddress, bridgeAbi) {
 }
 
 async function main() {
-  const { COMMON_HOME_BRIDGE_ADDRESS, COMMON_FOREIGN_BRIDGE_ADDRESS } = process.env
+  const {
+    COMMON_HOME_BRIDGE_ADDRESS,
+    COMMON_FOREIGN_BRIDGE_ADDRESS,
+    ORACLE_HOME_START_BLOCK,
+    ORACLE_FOREIGN_START_BLOCK
+  } = process.env
 
   const homeRpcUrl = rpcUrlsManager.homeUrls[0]
   const foreignRpcUrl = rpcUrlsManager.foreignUrls[0]
-  const homeStartBlock = await getStartBlock(homeRpcUrl, COMMON_HOME_BRIDGE_ADDRESS, homeABI)
-  const foreignStartBlock = await getStartBlock(foreignRpcUrl, COMMON_FOREIGN_BRIDGE_ADDRESS, foreignABI)
+  const homeStartBlock =
+    Number(ORACLE_HOME_START_BLOCK) || (await getStartBlock(homeRpcUrl, COMMON_HOME_BRIDGE_ADDRESS, homeABI))
```

Comment:
do you think we still need to check that the variables already defined even if we have changes in `deployment/roles/oracle/tasks/post_config.yml`? I think `getValidatorStartBlocks.js` should return the number of blocks where the contracts were deployed even the variables are already defined. Because, for example, they could be defined by incorrect values.

Reply:
Right, yes I think it makes sense to keep the previous behavior in `getValidatorStartBlocks.js`. I'll revert the changes in this file

Reply:
Updated here https://github.com/poanetwork/tokenbridge/pull/235/commits/1f16589989266caba990bd632c3de0aa34d8d32f